lib/licenses.nix

```
@@ -546,6 +546,18 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
     fullName = "Non-Profit Open Software License 3.0";
   };
 
+  nvidia_cuda = {
```

```suggestion
  nvidiaCuda = {
```
Caml case is the default naming scheme in nix/nixpkgs. Is there a particular motivation to deviate here?

---

saw more licenses with `-` and `_` than camel but happy to change